front/app/containers/Admin/index.tsx

```diff
@@ -174,14 +178,23 @@ const AdminPage = memo<Props & WithRouterProps>(
       >
         <ThemeProvider theme={chartTheme}>
           <Container className={`${className} ${whiteBg ? 'whiteBg' : ''}`}>
-            <Sidebar />
-            <RightColumn
-              className={`${fullWidth && 'fullWidth'} ${
-                noPadding && 'noPadding'
-              }`}
-            >
-              {children}
-            </RightColumn>
+            {!contentBuilderLayoutVisible && (
+              <>
+                <Sidebar />
+                <RightColumn
+                  className={`${fullWidth && 'fullWidth'} ${
+                    noPadding && 'noPadding'
+                  }`}
+                >
+                  {children}
+                </RightColumn>
+              </>
+            )}
+            <Outlet
```

Comment:
The outlet renders `null` if the feature flag is not active or the route is not page-builder